src/export_vdf.py

```diff
@@ -99,6 +100,34 @@ def export_qdrant(args):
     qdrant_export.get_data()
     return qdrant_export
 
+def export_kdbai(args):
+    """
+    Export data from KDBAI
+    """
+    set_arg_from_input(
+        args,
+        "url",
+        "Enter the KDB.AI endpoint instance: ",
+        str,
+    )
+    set_arg_from_input(
+        args,
+        "tables",
+        "Enter the name of table to export:",
+        str,
+    )
```

Comment:
would be good to show a list of choices here. probably need top prompt for api key before this and initialize a client object based on the endpoint instance info